### ISSUE TITLE
TRG 1.02 - an appropriate `INSTALL.md`

### ISSUE THREAD

**Issue by FaGru3n** [eclipse-tractusx/daps-helm-chart#24]
Genreal File is available, but some needed things are missing:

- local installation Markdown remove the html coded like  <br>
![image](https://github.com/eclipse-tractusx/daps-helm-chart/assets/121097161/ca05c433-8cd9-404a-aea0-1441d480febe)

- copy/ code section should be improved

a) Add helm repository in tractusx

``` bash
helm repo add daps https://eclipse-tractusx.github.io/charts/dev
``` 

> b) To search the specific repo in helm repositories

``` bash
helm search repo daps/daps-server
``` 

- how to do would be nice if an not an expirienced user will use the chart?
d.) These secrets should be defined in Hashicorp vault. <br />
e.) Deploy in a kubernetes cluster  <br />

-  basically here you can describe all nessesarry things you need to enable or provide the bring the chart running and how you should use your application.

**Comment by adkumar1** [comment]
Removed html code.

Same steps has been followed for all the repositories. And its working perfectly fine

**Comment by Siegfriedk** [comment]
@adkumar1 the point is still correct though. When you do a local installation how do i get the secret?

**Comment by adkumar1** [comment]
No need to define the secret. it will take default values from secret.yaml 

  ClientID: {{ .Values.daps.secret.clientId | default (randAlphaNum 16) }}
  ClientSecret: {{ .Values.daps.secret.clientSecret | default (randAlphaNum 16) }}

**Comment by adkumar1** [comment]
Also updated with 807d47fd2ba0eb6890c8594d91412582a139e7be

**Comment by Siegfriedk** [comment]
not perfect but hint was added
